app/modules/individuals/parameters.py

```diff
@@ -26,29 +29,67 @@ class Meta(schemas.DetailedIndividualSchema.Meta):
 
 class PatchIndividualDetailsParameters(PatchJSONParameters):
     # pylint: disable=abstract-method,missing-docstring
-    OPERATION_CHOICES = (PatchJSONParameters.OP_REPLACE,)
+    OPERATION_CHOICES = (
+        PatchJSONParameters.OP_REPLACE,
+        PatchJSONParameters.OP_ADD,
+        PatchJSONParameters.OP_REMOVE,
+    )
+
+    # Is this weird? I need to modify EDM and Houston records
+    PATH_CHOICES_HOUSTON = ['encounters']
```

Comment:
something very wrong here. These shouldn't be lists, they should be sets. See encounters/parameters.py. 
My trick is to put a breakpoint here:
https://github.com/WildMeOrg/houston/blob/main/flask_restx_patched/parameters.py#L115
and then look at self.PATH_CHOICES